sdk/data/stream/stream_conn.go

```diff
@@ -185,3 +192,18 @@ func sortByStatus(dp *wrapper.DataPartition, selectAll bool) (hosts []string) {
 
 	return
 }
+
```

Comment:
If the machine is turned on and down, the status is false, and after 1 minute, it becomes true again.. Here, getNearHost is called. If the first two copies are not available, will the machine be read eventually?

Reply:
yes,  the machine who are not available will be try read after normal status machine